public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -0,0 +1,237 @@
+include ../_util-fns
+
+:marked
+  # Http
+  
+  Our application has become a huge success and our stakeholders have already expanded the vision to include integration with a hero api. 
+  
+  The current solution limits us to a fixed set of heroes, but integration with the api will allow us to load heroes dynamically from a database. We will also be able to add, edit and delete heroes. 
+  
+  In this chapter we will show how to integrate the http based api with our existing application.
+:marked
+  [Run the live example](/resources/live-examples/toh-6/ts/plnkr.html).   
+  
+.l-main-section
+:marked
+  ## Where We Left Off
+  Before we continue with our Tour of Heroes, let’s verify that we have the following structure after adding our hero service 
+  and hero detail component. If not, we’ll need to go back and follow the previous chapters.
+
+.filetree
+  .file angular2-tour-of-heroes
+  .children
+    .file app
+    .children
+      .file app.component.ts
+      .file app.component.css
+      .file dashboard.component.css
+      .file dashboard.component.html
+      .file dashboard.component.ts
+      .file hero.ts
+      .file hero-detail.component.css
+      .file hero-detail.component.html
+      .file hero-detail.component.ts
+      .file hero.service.ts
+      .file heroes.component.css
+      .file heroes.component.html
+      .file heroes.component.ts
+      .file main.ts
+      .file mock-heroes.ts
+    .file node_modules ...
+    .file typings ...      
+    .file index.html
+    .file package.json
+    .file styles.css
+    .file tsconfig.json
+    .file typings.json
+:marked
+  ### Keep the app transpiling and running
+  Open a terminal/console window and enter the following command to
+  start the TypeScript compiler, start the server, and watch for changes:
+
+code-example(format="." language="bash").
+  npm start
+
+:marked
+  The application runs and updates automatically as we continue to build the Tour of Heroes.
+
+.l-main-section
+:marked
+  ## Heroes and Http
+  
+  Let's look back at our `HeroService` implementation in the <a href="/docs/ts/latest/tutorial/toh-pt4.html">services</a> chapter. Specifically let's revisit the point we made about using promises when loading heroes. 
```

Comment:
I love this "callbacks" to previous content as a recap but about this one... I guess that the user will have this `HeroService` open from his work through the tutorial so he can see the function you are talking about. Problem I see is that the link sends you to the entire document instead of showing the concrete piece you are talking about.

Personally I would give a better link or simply +makeExample using the toh-4 source and showing it directly.

Thoughts?